plugin/server/metaRunners/MRPLUGIN_UPackPack.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    https://inedo.com/support/documentation/proget/reference/upack
+
+   upack pack «source» 
+        [\-\-targetDirectory=«targetDirectory»]
+        [\-\-group=«group»] 
+        [\-\-name=«name»]
+        [\-\-version=«version»]
+        [\-\-title=«title»] 
+        [\-\-description=«description»]
+        [\-\-icon=«icon»]
+
+        source - Directory containing files to add to the package.        
+        targetDirectory - Directory where the .upack file will be created. If not specified, the current working directory is used. (optional)
+        group - Package group. 
+        name - Package name. 
+        version - Package version. 
+        title - Package title. (optional)
+        description - Package description. (optional)
+        icon - Icon absolute Url.  (optional)
+-->
+<meta-runner name="UPack Pack">
+    <description>Run UPack Pack to create a ProGet Universal Feed package.</description>
+    <settings>
+        <parameters>
+            <param name="teamcity.tool.upack.source"
+                   value=""
+                   spec="text description='Directory containing files to add to the package.' label='Source' validationMode='not_empty' display='normal'" />
+            <param name="teamcity.tool.upack.targetDirectory"
+                   value=""
+                   spec="text description='Directory where the .upack file will be created. If not specified, the current working directory is used.' label='Target Directory' validationMode='any' display='normal'" />
+            <param name="teamcity.tool.upack.group"
+                   value=""
+                   spec="text description='Package group.' label='Group' validationMode='not_empty' display='normal'" />
+            <param name="teamcity.tool.upack.name"
+                   value=""
+                   spec="text description='Package name.' label='Package Name' validationMode='not_empty' display='normal'" />
+            <param name="teamcity.tool.upack.title"
+                   value=""
+                   spec="text description='Package title. (optional)' label='Pacakge Title' validationMode='any' display='normal'" />
+            <param name="teamcity.tool.upack.description"
+                   value=""
+                   spec="text description='Package description. (optional)' label='Package Description' validationMode='any' display='normal'" />
+            <param name="teamcity.tool.upack.version"
+                   value=""
+                   spec="text description='Package version. (only 3 positions are allowed. i.e.: #.#.#, but not #.#.#.#)' label='Version' validationMode='not_empty' display='normal'" />
```

Comment:
I wonder if we can be smarter about the validation here, use a regex or something. I found [one over on this Node project that may be of use](https://github.com/sindresorhus/semver-regex/blob/master/index.js#L2). I'm guessing you could also describe the version as "Package semantic version." since semver defines that only 3 positions are allowed.

Reply:
That makes sense.  I'll make the change.

Reply:
Cool. Make sure to test it; I'm on holiday right now, so if you can't test it I won't have access to my TeamCity setup until 2018. 😄